presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -171,6 +171,13 @@ Accessors
     Singular geometries (e.g., POINT, LINESTRING, POLYGON), are treated as collections of one element.
     Empty geometries are treated as empty collections.
 
+.. function:: ST_PointN(LineString, index) -> Point
```

Comment:
perhaps, update documentation for `ST_StartPoint` and `ST_EndPoint` to refer to this function

* ST_StartPoint: This is a shortcut for `ST_PointN(geometry, 1)`
* ST_EndPoint: This is a shortcut for `ST_PointN(geometry, ST_NumPoints(geometry))`